update Observability diagrams (#11582)
kyma-project/kyma: docs/05-technical-reference/03-architecture/obsv-01-architecture-monitoring.md

## End-to-end monitoring flow

The monitoring flow in Kyma comes down to the following components and steps:

> ![End-to-end monitoring flow](./assets/obsv-monitoring-flow.svg)

Reviewer: The text on the arrow from operator to alertmanager is missing. It should be the same text as on the other arrow

Author: Thanks for spotting it! Done

> 1. Upon Kyma installation on a cluster, **Prometheus Operator** creates a **Prometheus** instance with the default configuration.

Reviewer: ```suggestion
1. Upon Kyma installation on a cluster, **Prometheus Operator** creates a **Prometheus** and a **Alertmanager** instance with the default configuration.
```

![Monitoring components](./assets/obsv-monitoring-architecture.svg)

> 1. [**Prometheus Operator**](https://github.com/coreos/prometheus-operator) creates a **Prometheus** instance, manages its deployment, and provides configuration for it. It also deploys **Alertmanager** and operates **ServiceMonitor** custom resources that specify monitoring definitions for groups of services.

Reviewer: One would operate an active component only. Here it is a resource, so managing is the better term
```suggestion
1. [**Prometheus Operator**](https://github.com/coreos/prometheus-operator) creates a **Prometheus** instance, manages its deployment, and provides configuration for it. It also deploys **Alertmanager** and manages **ServiceMonitor** custom resources that specify monitoring definitions for groups of services.
```

> 2. [**Prometheus**](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. To obtain such metrics, Prometheus uses the [**kube-state-metrics**](https://github.com/kubernetes/kube-state-metrics) service. It generates the metrics from Kubernetes API objects and exposes them on the `/metrics` **HTTP endpoint**.  

Reviewer: ```suggestion
2. [**Prometheus**](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. Metrics specific to applications (like orders processed per second) are polled from the application pods. Metrics common for all applications are scraped from specialized pods like the **node-exporter**. 
```

> Pods can also contain applications with custom metrics, such as the total storage space available in the MinIO server. Prometheus stores this polled data in a time-series database (TSDB) and runs rules over them to generate alerts if it detects any metric anomalies. It also scrapes metrics provided by [**Node Exporter**](https://github.com/mindprince/nvidia_gpu_prometheus_exporter) which exposes existing hardware metrics from external systems as Prometheus metrics.

Reviewer: ```suggestion
Prometheus stores this polled data in a time-series database (TSDB) and runs rules over them to create aggregated metrics or generate alerts if it detects any metric anomalies. You can use **PrometheusRules** to define alert conditions for metrics. Kyma provides a set of out-of-the-box alerting rules. The definitions of such rules specify the alert logic, the value at which alerts are triggered, the alerts' severity, and more.
```

> 4. [**Alertmanager**](https://prometheus.io/docs/alerting/alertmanager/) receives alerts from Prometheus and forwards this data to configured Slack or Victor Ops channels.  You can use **PrometheusRules** to define alert conditions for metrics. Kyma provides a set of out-of-the-box alerting rules that are passed from Prometheus to Alertmanager. The definitions of such rules specify the alert logic, the value at which alerts are triggered, the alerts' severity, and more.

Reviewer: ```suggestion
4. [**Alertmanager**](https://prometheus.io/docs/alerting/alertmanager/) receives alerts from Prometheus and forwards this data to configured notification channels like Slack or Victor Ops. 
```
